### PR TITLE
{bio}[foss/2021b] AlphaFold v2.3.0 w/ Python 3.9.6 + CUDA 11.4.1

### PR DESCRIPTION
**Pull request by ThomasHoffmann77**
(created using `eb --new-pr`)
requires:
* #16870
* #16871

### CODE REVIEW

**Comment by nortex** [comment]
Are there any estimates when AlphaFold 2.3.0 / 2.3.1 will be available as easyconfig?

**Comment by boegel** [comment]
@nortex I'm actively looking into getting this PR merged now...
I'll try and also look into AlphaFold 2.3.1 (probably using `foss/2022a`)

**Comment by nortex** [comment]
> @nortex I'm actively looking into getting this PR merged now... I'll try and also look into AlphaFold 2.3.1 (probably using `foss/2022a`)

Also for CUDA support?

**Comment by boegel** [comment]
> > @nortex I'm actively looking into getting this PR merged now... I'll try and also look into AlphaFold 2.3.1 (probably using `foss/2022a`)
> 
> Also for CUDA support?

CUDA support is important for AlphaFold, so that's the goal, yes

File: easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.0-foss-2021b-CUDA-11.4.1.eb [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+        'checksums': ['3fefcc72ec433aa1e5d32307a3e474bbb67f405be814ea52a2166bfc9dbe68cc'],
+    }),
+]
+
+postinstallcmds = [
+    "mkdir -p %(installdir)s/bin",
+    # run_alphafold.py script is missing a shebang...
+    "echo '#!/usr/bin/env python' > %(installdir)s/bin/run_alphafold.py",
+    "cat %(builddir)s/alphafold-%(version)s/run_alphafold.py >> %(installdir)s/bin/run_alphafold.py",
+    "cp %(builddir)s/alphafold-%(version)s/run_alphafold_{features,resume}.py %(installdir)s/bin",
```
Comment (boegel) [inline easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.0-foss-2021b-CUDA-11.4.1.eb:134]: @ThomasHoffmann77 This looks wrong to me, where are `run_alphafold_features.py` and `run_alphafold_resume.py` supposed to come from?

Comment (ThomasHoffmann77) [reply easybuild/easyconfigs/a/AlphaFold/AlphaFold-2.3.0-foss-2021b-CUDA-11.4.1.eb:134]: we additionally provide _features and _resume in order to run features generation on CPU nodes. I missed to delete this line.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @ThomasHoffmann77!

**Pull request merged**